src/status_im2/subs/general.cljs

```diff
@@ -228,12 +228,6 @@
  (fn [network]
    (tokens/native-currency network)))
 
-(re-frame/reg-sub
- :information-box-closed?
- :<- [:information-box-states]
- (fn [states [_ id]]
-   (get states id)))
-
```

Comment:
hi, are we removing information box states?

Reply:
yes ,they are not used

Reply:
no problem to restore them later when needed